src/app/modules/users/models/users.ts

```diff
@@ -2,6 +2,7 @@ export interface User {
   name: string;
   email: string;
   role?: string;
+  roles?: string[];
```

Comment:
Where are the tests for this? I mean, when we saw the implementation you showed me a simulation of FF so you used a mocked users json, you could test in the same way by mocking the response of the endpoint and what the model would receive. Is it that possible?

Reply:
Sure, I have added a test to check it